src/main/java/world/bentobox/bentobox/api/configuration/WorldSettings.java

```diff
@@ -467,4 +467,61 @@ default boolean isTeleportPlayerToIslandUponIslandCreation() {
         return true;
     }
 
+
+    /**
+     * Returns all aliases for main admin command.
+     * It is assumed that all aliases are split with whitespace between them.
+     * String cannot be empty.
+     * Default value: {@code getFriendlyName() + "admin"} (to retain backward compatibility).
+     * @return String value
+     * @since 1.13.0
+     */
+    default String getAdminCommandAlias()
+    {
+        return this.getFriendlyName().toLowerCase() + "admin";
+    }
+
+
+    /**
+     * Returns all aliases for main user command.
+     * It is assumed that all aliases are split with whitespace between them.
+     * String cannot be empty.
+     * Default value: {@code getFriendlyName()} (to retain backward compatibility).
+     * @return String value
+     * @since 1.13.0
+     */
+    default String getUserCommandAlias()
```

Comment:
```suggestion
    default String getIslandCommandAliases()
```

I think it's "more" understandable this way.